tests/integration/Help_Hub/__snapshots__/Template_Test__overall_template__0.snapshot.html

```diff
@@ -1,6 +1,6 @@
 <div class="tribe_settings wrap tec-events-admin-settings">
 	<div class="tribe-notice-wrap">
-	<div class="wp-header-end"></div>
+		<div class="wp-header-end"></div>
 </div>
```

Comment:
Does this need to be indented too?
```suggestion
	</div>
```